lupa/_lupa.pyx

```diff
@@ -50,6 +50,11 @@ try:
 except ImportError:
     from collections import Mapping  # Py2
 
+try:
+    from sys import maxint
+except ImportError:
+    maxint = None
```

Comment:
`cdef long maxint`, then set `maxint = 0` instead of `None` in Py3.

Reply:
That's not present in the current version of _lupa.pyx. You can safely ignore this and squash all commits into one.